latex/Research_Strategy.tex

```diff
@@ -190,6 +190,23 @@
 
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%SAMPL HISTORY
+\begin{wrapfigure}[20]{r}{7cm}
+%\vspace{-0.25in}
+\begin{centering}
+\includegraphics[width=\textwidth]{figures/history_v1.pdf}
+
+\end{centering}
+\footnotesize
+\caption{\label{figure:sampl5_logD}  
+\textbf{SAMPL historical participation~\cite{mobley_blind_2014-1}.} 
+Historical participation in SAMPL host-guest + solvation/distribution challenges has climbed rapidly, and we expect this trend to continue. The number of participating groups is shown in blue, and the number of submissions in green. The inset shows the diversity method types employed for the SAMPL4 hydration challenge, which is typical for SAMPL. 
```

Comment:
"diversity method types" -> "diversity of method types"

I really like this figure!

On my printer, the pie chart text key is slightly too small, and the blue and green look almost the same color. If easy to fix would be nice, but not critical.